indicators/views/bulk_indicator_import_views.py

```diff
@@ -96,7 +100,29 @@
 logger = logging.getLogger('__name__')
 
 class BulkImportIndicatorsView(LoginRequiredMixin, UserPassesTestMixin, AccessMixin, View):
-    """Returns bulk import .xlsx file"""
+    """
+    This view is used for the template download and upload process.
+
+    For the download process, it checks the request to see how many rows each tier is configured for and then returns
+    a template with the appropriate structure and fillable row numbers.
+
+    For the upload process, it runs though lots of checks to validate the data.  If the validation passes, a json
+    file is kept on the file system and a new instance  of the BulkIndicatorImportFile that points to the json
+    file.  When the user confirms that they want to save the indicators, a different view is used to retrieve the
+    json file and create the Indicator model instances.
+
+    If the validation checks fail, the errors are divded into fatal and non-fatal categories.  Non-fatal errors are
```

Comment:
`divided` has a typo

Reply:
Good spot, thanks.  I'm still working on this file so I'll fix it in the next PR.